src/Storage/Capabilities.php

```diff
@@ -162,6 +162,7 @@ public function getAdapter()
     /**
      * Get supported datatypes
      *
+     * @psalm-suppress MixedReturnStatement
```

Comment:
Please can you drop this inline suppression?

It's better off in the baseline, and it'll mean fewer conflicts when merging up into 4.x

Reply:
Done with e099a96